### PR TITLE
Builder/korolchuk

### PR DESCRIPTION
**Pull request by yurii-korolchuk**
Create builder example

### CODE REVIEW

File: Korolchuk/builder/src/Engine.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+public class Engine {
+    private final double volume;
+    private double mileage;
+    private boolean started;
+
```
Comment (VladKriklya) [inline Korolchuk/builder/src/Engine.java:5]: Good

File: Korolchuk/builder/src/Engine.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    public void off() {
+        started = false;
+    }
+
+    public boolean isStarted() {
+        return started;
+    }
+
+    public void go(double mileage) {
```
Comment (Yeuhen-Dubliak) [inline Korolchuk/builder/src/Engine.java:23]: Good function)

**Review by VladKriklya — COMMENTED** [review]
Okey

File: Korolchuk/builder/src/Engine.java (hunk 2 above)
Comment (yurii-korolchuk) [reply Korolchuk/builder/src/Engine.java:23]: Thatnks a lot

**Review by Yeuhen-Dubliak — COMMENTED** [review]
Good code, no errors

**Pull request merged**